api/spec/v1.yaml

```diff
@@ -1356,6 +1362,17 @@ components:
       description: |
         A consensus block.
 
+    NodeInfo:
+      type: object
+      properties:
+        node_id:
```

Comment:
`node_id` is not needed to satisfy the consensus mocks, let's leave it out since it simplifies things. (Plus, i doubt most users would be able to use it, `entity_id` is more helpful since it corresponds to the validator)